hi-ml-azure/testazure/testazure/test_azure_util.py

```diff
@@ -1891,6 +1893,27 @@ class SimpleClass(param.Parameterized):
     assert "my_epilog" in stdout
 
 
+@pytest.mark.fast
+@pytest.mark.parametrize("args, expected_key, expected_value, expected_pass", [
+    (["--strings=[]"], "strings", ['[]'], True),
+    (["--strings=['']"], "strings", ["['']"], True),
+    (["--strings=None"], "strings", ['None'], True),
+    (["--strings='None'"], "strings", ["'None'"], True),
+    (["--strings=','"], "strings", ["'", "'"], True),
+    (["--strings=''"], "strings", ["''"], True),
+    (["--strings=,"], "strings", [], True),
+    (["--strings="], "strings", [], True),
+    (["--integers="], "integers", [], True),
+    (["--floats="], "floats", [], True)])
+def test_override_list(parameterized_config_and_parser: Tuple[ParamClass, ArgumentParser],
+                       args: List[str], expected_key: str, expected_value: Any, expected_pass: bool) -> None:
+    """Test different options of overriding a non-empty list parameter to get an empty list"""
```

Comment:
nitpick: All your expected_pass settings are True, why have that parameter?

Reply:
Seconding this: either remove the `expected_pass` variable or add expected failing tests (however for this specific test I don't think that's necessary as the failing tests seem to be covered by `test_create_parser()`

Reply:
Good point, removed